### PR TITLE
T13152

### PR DESCRIPTION
**Pull request by tchx84**
https://phabricator.endlessm.com/T13152

### CODE REVIEW

File: data/preset/news.yaml [hunk 1]
```diff
@@ -155,10 +155,7 @@ defines:
                               slots:
                                 order: 'Order.PublishedDate(ascending: false)'
                                 filter:
-                                  type: Filter.OtherGroup
```
Comment (rmacqueen) [inline data/preset/news.yaml]: `Filter.OtherGroup` isn't used at all anymore, right? So we can just kill it

**Comment by rmacqueen** [comment]
Just a comment that I think you can now remove some dead code. Other than that looks good

**Comment by rmacqueen** [comment]
CR PASS

**Comment by ptomato** [comment]
We definitely wouldn't remove Filter.OtherGroup on the eos3.0 branch, that would be an API break, and it would break apps if people hadn't updated recently. We probably should not remove any modules until we have a good story for releasing new eos-knowledge runtime versions.

Maybe we could add a comment to Filter.OtherGroup saying it's being considered for removal, to remind ourselves to review it next time we have an API break? Deprecated would be a bit strong, since if we found a good use for it in the meantime we would still keep it.

**Pull request merged**